.github/workflows/test.yml

```diff
@@ -98,18 +91,29 @@ jobs:
             --name kyma
       - name: Update Kubeconfigs
         run: k3d kubeconfig merge -a -d
-      - name: Run Deploy
+      - name: Setup Control-Plane requirements
+        if: ${{ matrix.target }} == 'control-plane'
         run: |
-          kyma --ci alpha deploy \
-            -k $(go env GOPATH)/src/github.com/$LIFECYCLE_MANAGER/config/default
-      - name: Inject GOPATH into go Setup
-        run: echo "GOPATH=$(go env GOPATH)" >> $GITHUB_ENV
-      - name: Set up Go for ${{ matrix.smoke-run }}
+          curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+          chmod +x kubectl
+          mv kubectl /usr/local/bin
+          
+          curl -L https://istio.io/downloadIstio | TARGET_ARCH=x86_64 sh -
```

Comment:
do we need to provide an ISTIO_VERSION here? as described in the this downloadIstio script
```
 curl -L https://istio.io/downloadIstio | ISTIO_VERSION=1.6.8 TARGET_ARCH=x86_64 sh -.
```

Reply:
we should also deploy cert manager, or? The problem we faced in DEV is because the istio sidecar blocked the requestion coming from apiserver for the admission webhook, right?

Reply:
the cert manager is already deployed in kyma alpha deploy.
the ISTIO_VERSION is already provided from the environment variables so you can observe that in the logs